src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -157,22 +164,27 @@ define(function (require) {
     };
 
     SegmentedReq.prototype._createQueue = function () {
+      var self = this;
```

Comment:
I feel like we should be setting `this._queueCreated = false;` here, and `this._queueCreated = true;` below by `self._queue = ...`;